cleanlab/rank.py

```diff
@@ -28,16 +28,20 @@
 
 
 import numpy as np
+from typing import List
 
 
 def order_label_issues(
-    label_issues_mask,
-    labels,
-    pred_probs,
+    label_issues_mask: np.array,
+    labels: np.array,
+    pred_probs: np.array,
     *,
-    rank_by="normalized_margin",
-):
-    """Sorts label issues by normalized margin.
+    rank_by: str = "normalized_margin",
+    rank_by_kwargs: dict = {},
+) -> np.array:
+    """Sorts label issues by label quality score.
+
+    Default label quality score is "normalized margin".
     See https://arxiv.org/pdf/1810.05369.pdf (eqn 2.2)
     e.g. normalized_margin = prob_label - max_prob_not_label
```

Comment:
Let's delete: "e.g. normalized_margin = prob_label - max_prob_not_label" and try to reduce redundancy between the docstrings now that there are many more of them.

Reply:
Done: 100783d9199110a66737f2d09e2fce762b500c45

Thanks